pytest/lib/mocknet.py

```diff
@@ -440,84 +440,68 @@ def compress_and_upload(nodes, src_filename, dst_filename):
-            validator_node_names = [
-                node.instance_name for node in validator_nodes
-            ]
-            rpc_node_names = [node.instance_name for node in rpc_nodes]
-            create_genesis_file(validator_node_names,
-                                genesis_template_filename,
-                                mocknet_genesis_filename,
-                                tmp_dir=tmp_dir,
-                                rpc_node_names=rpc_node_names,
-                                epoch_length=epoch_length,
-                                node_pks=node_pks,
-                                increasing_stakes=increasing_stakes,
-                                num_seats=num_seats)
-            # Save time and bandwidth by uploading a compressed file, which is 2% the size of the genesis file.
-            compress_and_upload(validator_nodes + rpc_nodes,
-                                mocknet_genesis_filename,
-                                '/home/ubuntu/.near/genesis.json')
-        else:
-            logger.info(
-                'Assuming that genesis_updater.py is available on the instances.'
-            )
-            validator_node_names = [
-                node.instance_name for node in validator_nodes
-            ]
-            rpc_node_names = [node.instance_name for node in rpc_nodes]
-            assert '-spoon' in chain_id, f'Expecting chain_id like "testnet-spoon" or "mainnet-spoon", got {chain_id}'
-            chain_id_in = chain_id.split('-spoon')[0]
-            genesis_filename_in = f'/home/ubuntu/.near/genesis.{chain_id_in}.json'
-            done_filename = f'/home/ubuntu/genesis_update_done_{int(time.time())}.txt'
-            pmap(
-                lambda node: start_genesis_updater(
-                    node, 'genesis_updater.py', genesis_filename_in,
-                    '/home/ubuntu/.near/genesis.json', chain_id,
-                    validator_node_names, rpc_node_names, done_filename,
-                    epoch_length, node_pks, increasing_stakes, num_seats),
-                validator_nodes + rpc_nodes)
-            pmap(lambda node: wait_genesis_updater_done(node, done_filename),
-                 validator_nodes + rpc_nodes)
+        assert update_genesis_on_machine
+        logger.info(
+            'Assuming that genesis_updater.py is available on the instances.')
+        validator_node_names = [node.instance_name for node in validator_nodes]
+        rpc_node_names = [node.instance_name for node in rpc_nodes]
+        assert '-spoon' in chain_id, f'Expecting chain_id like "testnet-spoon" or "mainnet-spoon", got {chain_id}'
+        chain_id_in = chain_id.split('-spoon')[0]
+        genesis_filename_in = f'/home/ubuntu/.near/${chain_id_in}/genesis.json'
+        records_filename_in = f'/home/ubuntu/.near/${chain_id_in}/records.json'
+        config_filename_in = f'/home/ubuntu/.near/${chain_id_in}/config.json'
```

Comment:
Do you mean to have `$` in the resulting string? As in, is this meant to be expanded with environment variables?